Tensile/Common.py

```diff
@@ -1384,14 +1395,17 @@ def getArchitectureName(gfxName):
     {"LocalDotLayout":            [ 1 ] },
     {"AggressivePerfMode":        [ 1 ] },
     {"KernelLanguage":            [ "Source" ] },
-    {"LdsPadA":                   [ 0 ] },
-    {"LdsPadB":                   [ 0 ] },
-    {"LdsBlockSizePerPad":        [ 0 ] },
+    {"LdsPadA":                   [ -1 ] },
+    {"LdsPadB":                   [ -1 ] },
+    {"LdsBlockSizePerPadA":       [ -1 ] },
+    {"LdsBlockSizePerPadB":       [ -1 ] },
```

Comment:
This splits LdsBlockSizePerPad to two parameters which have a different default than the original - is this going to affect existing rocblas tuning? Have you run rocblas tests?

Reply:
Mostly not because most of existing kernels already have separate values in logic files.
If some very old kernels do not have these parameters, it can be affected.
I have not run rocblas-test yest (I confirmed rocblas build).
I can run rocblas-test with this change.

By the way, this default change revealed a lot of existing bugs with current Tensile precheckin+extended tests.

Reply:
Yea, definitely worth running rocblas-test if Tensile bugs popped up. Thanks!